code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -311,6 +311,30 @@
 	for(var/Z in limb_list)
 		. += regenerate_limb(Z, noheal)
 
+/mob/living/proc/regenerate_organs()
```

Comment:
not sure if this belongs in this file but I can't think of a better place for it so whatever